cluster/node-pools/master-default/userdata.yaml

```diff
@@ -41,11 +41,9 @@ write_files:
 {{- if not (index .Cluster.ConfigItems "enable_cfs_quota") }}
       cpuCFSQuota: false
 {{- end }}
-{{- if ne .NodePool.ConfigItems.pod_max_pids "-1" }}
```

Comment:
I dropped this as it's now enabled by default and simplified the configuration here.